app/src/main/java/org/gdg/frisbee/android/fragment/FirstStartStep1Fragment.java

```diff
@@ -138,7 +138,6 @@ public void onNotFound(String key) {
             @Override
             public void onClick(View view) {
                 Chapter selectedChapter = (Chapter) mChapterSpinner.getSelectedItem();
-                getArguments().putParcelable("selected_chapter", selectedChapter);

```

Comment:
Arguments are never used elsewhere in the Fragment. Selected item in the Spinner will probably reset when device is rotated. We can make a separate issue for that. 
And for state restoration, savedInstanceState should be used. Using Arguments for that is wrong.